src/components/NavBar/NavBar.tsx

```diff
@@ -105,18 +103,18 @@ const NavBar: React.FunctionComponent = () => {
           Woohoo, youv&apos;e completed today&apos;s challenges!
         </Modal.Body>
         <Modal.Footer>
-          <Button variant="secondary" onClick={handleCloseCompleted}>
+          <Button className={styles.dcmodalBtn} onClick={handleCloseCompleted}>
             Nice!
           </Button>
-          <Button variant="primary" onClick={handleCloseCompleted}>
+          <Button className={styles.dcmodalBtn} onClick={handleCloseCompleted}>
             View challenges
```

Comment:
`View Challenges` -> `View LeaderBoard` . We can redirect him to `daily-challenge leaderboard`.
_(DC-LeaderBoard is not yet ready, you can add the calling latter)_